app/uk/gov/hmrc/helptosave/controllers/EnrolmentBehaviour.scala

```diff
@@ -18,25 +18,34 @@ package uk.gov.hmrc.helptosave.controllers
 
 import cats.data.EitherT
 import cats.instances.future._
-import uk.gov.hmrc.helptosave.connectors.ITMPEnrolmentConnector
 import uk.gov.hmrc.helptosave.models.register.CreateAccountRequest
+import uk.gov.hmrc.helptosave.connectors.DESConnector
 import uk.gov.hmrc.helptosave.repo.EnrolmentStore
 import uk.gov.hmrc.helptosave.util._
 import uk.gov.hmrc.http.HeaderCarrier
+import play.api.http.Status
 
 import scala.concurrent.{ExecutionContext, Future}
 
 trait EnrolmentBehaviour {
 
   val enrolmentStore: EnrolmentStore
 
-  val itmpConnector: ITMPEnrolmentConnector
+  val desConnector: DESConnector
```

Comment:
why connector here . ? shouldnt it be `helptosaveservice` ?

Reply:
yes you're right